imu_publisher:
  ros__parameters:
    serial_port: "/dev/ttyUSB1"
    baud_rate: 115200
    frame_id: "imu_frame"
